zio-schema-derivation/shared/src/main/scala-3/zio/schema/DeriveSchema.scala

```diff
@@ -256,7 +256,7 @@ private case class DeriveSchema()(using val ctx: Quotes) {
                tpe.asType
            }
            val annotations = paramAnns.getOrElse(label, List.empty)
-           val nameExpr = annotations.collectFirst {
```

Comment:
I actually think more the one annotation of the same kind is using the lib wrong. I would keep the old impl, since it is less complicated. You annotate twice? Your problem. Just don't.

Reply:
The proposed solution is a workaround for reverse order of annotations in Scala 3 macros.

The better solution would be throwing a compile-time error during derivation or/and runtime error when calculating `fieldName` value in `Schema.Field` instances